library/src/main/scala/scala/scalajs/js/Object.scala

```diff
@@ -57,6 +57,37 @@ object Object extends Object {
   def apply(): Object = native
   def apply(value: Any): Object = native
 
+  /** <span class="badge badge-ecma6" style="float: right;">ECMAScript 6</span>
+   * The `Object.assign()` method is used to copy the values of all enumerable
```

Comment:
Ah, almost 😄 There should be one more space between the `*` and the text. That way, the first character of the text is always aligned, from the first line to the last one. So it starts as
```scala
  /** <span class="badge badge-ecma6" style="float: right;">ECMAScript 6</span>
   *  The `Object.assign()` method is used to copy the values of all enumerable
   *  own properties from one or more source objects to a target object. It will
   *  return the target object.
```
Other than that, good to squash and then we can merge it.

Reply:
Gotcha, fixed and squashed